src/Form/Field/ContentTypeFieldChoiceList.php

```diff
@@ -21,27 +28,31 @@ public function __construct(array $mapping, array $types, $firstLevelOnly)
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods